sspinn/version.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import absolute_import, division, print_function
+from os.path import join as pjoin
```

Comment:
Is this even used?

Reply:
I'm not sure what the `__future__` imports were doing; none of that functionality was used. `os.path.join` was used at the bottom of the file for specifying the data directory, but that doesn't yet exist for us.

I removed both the lines and am pushing again